### DISCUSSION TITLE
2023 team names not updated for football private league

### DISCUSSION THREAD

**Discussion by davidtenorio1** [cwendt94/espn-api#468] [ESPN API] [answered]
I have no trouble pulling any 2023 data other than team names. The team names are still showing for 2022. This is a private league. I've tried updating espn_s2 and swid, but that didn't resolve the issue. I double checked i have the correct league id and year. Team_abbrev is updated for 2023, just not team_name.

**Comment by dtcarls** [comment, marked as answer]
Sounds like you aren’t using the latest version. Which version is installed? This was fixed right before the season started.

**Reply by davidtenorio1** [reply]
Yup, I was using the older version. Updating fixed the issue for me. Thank you.

**Comment by davidtenorio1** [comment]
I had to update the espn-api package itself. i put this into the cmd
line:  *pip
install espn-api --upgrade*
